canonical-websites/blog.ubuntu.com
# Remove b variant of homepage

## Done

Remove b variant of homepage

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: <http://0.0.0.0:8023/home>
- See that you are redirected back to /


## Issue / Card

Fixes #382
